app/views/common/DescriptionTemplate.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import {
+  ImageBackground,
+  StyleSheet,
+  TouchableOpacity,
+  TextStyle,
+  View,
+  ScrollView,
+  ImageSourcePropType,
+  ViewStyle,
+} from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import { useStatusBarEffect } from '../../navigation';
+import { Typography } from '../../components/Typography';
+
+import {
+  Buttons,
+  Colors,
+  Layout,
+  Spacing,
+  Iconography,
+  Typography as TypographyStyles,
+} from '../../styles';
+
+type DescriptionTemplateProps = {
+  iconXml: string;
+  header: string;
+  headerStyle?: TextStyle;
+  body: string;
+  bodyStyle?: TextStyle;
+  primaryButtonLabel: string;
+  primaryButtonContainerStyle?: ViewStyle;
+  primaryButtonTextStyle?: TextStyle;
+  primaryButtonOnPress: () => void;
+  secondaryButtonLabel?: string;
+  secondaryButtonContainerStyle?: ViewStyle;
+  secondaryButtonTextStyle?: TextStyle;
+  secondaryButtonOnPress?: () => void;
+  backgroundImage: ImageSourcePropType;
+  backgroundStyle?: ViewStyle;
+  invertIcon?: boolean;
```

Comment:
I decided not to touch this, but the name seems wrong to me because setting this to true doesn't exactly "invert" the icon. Perhaps something like `useGoldIcon` would be more clear.